docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "59db66ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture \n",
+    "!pip install transformers tqdm "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e20de927",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# packages version: \n",
+    "# numpy == 1.23.0 \n",
+    "# tqdm == 4.64.0 \n",
+    "# transformers == 4.20.1 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Token Classification Label Error Detection - Part 1 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6676421",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we show how you can retrieve the model-predictive probabilities and labels from a NLP token-classification dataset. These inputs are used to identify potential label issues in the dataset. \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "- Read and process datasets in CoNLL formats \n",
+    "- Compute out-of-sample predicted probabilities via cross-validation \n",
+    "- Reduce subword-level tokens to word-level tokens \n",
+    "\n",
+    "\\* In most NLP literatures, tokens typically refer to words or punctuation marks, while most HuggingFace tokenizers break down longer words into subwords. To avoid confusion, given tokens are referred as \"word-level tokens\", and tokens obtained from the tokenizers are referred as \"subword-level tokens\". "
```

Comment:
"given tokens" probably needs to be clarified as something like:
"the individual tokens given in the original dataset (with a separate class label provided for each token)"

Reply:
HuggingFace tokenizers -> modern tokenizers